ProcessLib/UncoupledProcessesTimeLoop.cpp

```diff
@@ -126,6 +126,7 @@ struct SingleProcessData
     //! other members of this struct to their concrety types.
     NumLib::NonlinearSolverTag const nonlinear_solver_tag;
     NumLib::NonlinearSolverBase& nonlinear_solver;
+    bool is_nonlinear_solver_converged;
```

Comment:
tiny: `nonlinear_solver_converged` would also be a good name w/o "time" indication.